### DISCUSSION TITLE
How to exit a granted console session?

### DISCUSSION THREAD

**Discussion by moltar** [fwdcloudsec/granted#536] [Q&A] [answered]
I am coming from the `aws-vault`, where I could do:

```sh
aws-vault exec $PROFILE
```

And it would create an AWS STS session and open a new `$SHELL` session with the credentials.

But it seems like `assume` is doing something different, and a new shell session is not created.

So when I try to `exit` to clear the credentials, it just closes a term tab.

Is there a way to get `assume` to open a new shell?

Or what is the correct way to clear the credentials from my current shell?

Thanks.

**Comment by Eddie023** [comment, marked as answer]
Hi @moltar , there is a `assume --unset` flag that will clear all the environment variables configured by assume. Is that what you are looking for?

**Reply by moltar** [reply]
@Eddie023 yeah, that could work, thanks for the tip.

But would still love to get the old workflow back where I can enter & exit the environment :)
